### DISCUSSION TITLE
access dashboard remotely?

### DISCUSSION THREAD

**Discussion by 2533245542** [optuna/optuna-dashboard#154] [Q&A]
Hi,

I mainly run Optuna on the server and I was just able to use the optuna-dashboard on the server based on a search db file. Is there a way for me to view the dashboard remotely, just like jupyter-notebook?

**Comment by c-bata** [comment]
Hi. I'm not sure what a search db file is. But you can access the dashboard remotely like jupyter-notebook. If you need to change the hostname and/or port number, you can it like the following:

```
$ optuna-dashboard --host 0.0.0.0 --port 8000 sqlite:///db.sqlite3
```

**Comment by ghylander** [comment]
I have a question related to this.
Can I store the SQL server on a remote server, then have optuna send the data from the training machine to the remote server?
Then to start the optuna dashboard, I can launch the http server on my laptop, and pointe the SQL database to the remote server.
Is this achievanle?

**Reply by c-bata** [reply]
@ghylander Yes, you can.

**Reply by ghylander** [reply]
Great, thanks!
